aikau/src/main/resources/alfresco/forms/controls/css/MultiSelect.css

```diff
@@ -168,11 +168,11 @@
    }
    &--has-error {
       border-color: #c00;
-      outline: 1px solid #faa;
+      box-shadow: 0 0 1px 1px #ffc4c4;
```

Comment:
Anything special about these box-shadow additions? Different from the defaults LESS variables? Should these be new LESS variables.